src/main/java/seedu/address/logic/parser/SessionCommandParser.java

```diff
@@ -0,0 +1,38 @@
+package seedu.address.logic.parser;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.core.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_SESSION;
+
+import seedu.address.commons.core.index.Index;
+import seedu.address.logic.commands.AttendanceCommand;
+import seedu.address.logic.commands.SessionCommand;
+import seedu.address.logic.parser.exceptions.ParseException;
+import seedu.address.model.person.Session;
+
+/**
+ * Parses input command and creates a new SessionCommand object.
+ */
+public class SessionCommandParser implements Parser<SessionCommand> {
+
+    /**
+     * Parses the given {@code String} of arguments in the context of the SessionCommand
+     * and returns a SessionCommand object for execution.
+     * @throws ParseException if the user input does not conform the expected format
+     */
+    @Override
+    public SessionCommand parse(String args) throws ParseException {
+        requireNonNull(args);
+        ArgumentMultimap argumentMultimap = ArgumentTokenizer.tokenize(args, PREFIX_SESSION);
+        Index index;
+
+        try {
+            index = ParserUtil.parseIndex(argumentMultimap.getPreamble());
+        } catch (ParseException pe) {
+            throw new ParseException(String.format(MESSAGE_INVALID_COMMAND_FORMAT,
+                    AttendanceCommand.MESSAGE_USAGE), pe);
```

Comment:
You forgot to replace Attendance with Session here

Reply:
apologies n thx